app/src/main/res/values/strings.xml

```diff
@@ -93,4 +93,12 @@
     <string name="delete_cover_image">Delete cover image</string>
     <string name="address_search">Enter Address</string>
     <string name="request_book_successful">Request successfully sent</string>
+    <string name="accepted_button">scan to receive</string>
+    <string name="requested_button">waiting for acceptance</string>
+    <string name="browse_button">click to request</string>
+    <string name="borrowed_button">scan to return </string>
+    <string name="owned_requested_button">click to see requests</string>
+    <string name="owned_accepted_button">scan to hand over</string>
+    <string name="owned_borrowed_button">scan to receive</string>
```

Comment:
I think the better practice is give a name that describe the text instead of the component it is currently used in